### PR TITLE
THREESCALE-2081: Disabling click in arrows icon of mapping rules

### PR DESCRIPTION
**Pull request by damianpm**
**What this PR does / why we need it**:

 Arrows icon that indicates drag&drop feature within mapping rules should not be clickable

**Which issue(s) this PR fixes** 

https://issues.jboss.org/browse/THREESCALE-2081

**Verification steps** 

- Go to Integration > Configuration
- Add some mapping rules

### CODE REVIEW

**Comment by damianpm** [comment]
> It only works for the rules created, not for the ones rendered with ralis.

@didierofrivia Updated, try now

File: app/views/api/integrations/apicast/shared/_mapping_rules.html.slim [hunk 1]
```diff
@@ -59,3 +59,6 @@ javascript:
       jQuery('tr, td').disableSelection()
     })
   })
+  jQuery('.ui-sortable-handler').click(function (event) {
```
Comment (didierofrivia) [inline app/views/api/integrations/apicast/shared/_mapping_rules.html.slim]: Could you do it on `create` the sortable list?

**Comment by didierofrivia** [comment]
I meant on `create` event of the `sortable` list:

```
...
items: 'tr:not(#new-proxy-rule-template)',
create: function (event, ui) {
  jQuery(event.target).find('.ui-sortable-handler').click(function (event) { 
   event.preventDefault()
  }
},
start: function (event, ui) {
  jQuery.each(ui.item.children(), function (i, td) {
...
```

OR

if you decide to make it in the same coffee you are disabling the new ones, create a fn to reuse the selection of the sortable handler and preventDefault.

OR

An ultimate and easier alternative would have been replacing the anchor by a `span` node hehe. The colour would change because of styling, but might be worth a look, see what @thomasmaas think about this last one.

**Comment by thomasmaas** [comment]
@dd `span` element instead of `a` element? yes, i think it shouldn't have been an `a` element in the first place as it doesn't link to anything. Semantically it has no meaning so `span` or `div` are ok.

File: app/assets/stylesheets/provider/_links.scss [hunk 2]
```diff
@@ -23,6 +23,10 @@ table button.action {
   display: inline-block;
 }
 
+table a.ui-sortable-handler {
```
Comment (thomasmaas) [inline app/assets/stylesheets/provider/_links.scss]:
```suggestion
.ui-sortable-handler {
```

and if we choose it to no longer be `a` element it belongs in a different stylsheet.

File: app/assets/stylesheets/provider/_links.scss [hunk 3]
```diff
@@ -23,6 +23,11 @@ table button.action {
   display: inline-block;
 }
 
+.ui-sortable-handler { // scss-lint:disable SelectorFormat
+  color: $success-color;
+  cursor: move;
+}
+
```
Comment (thomasmaas) [inline app/assets/stylesheets/provider/_links.scss]: Let's move this to `commons.scss`?

**Pull request merged**